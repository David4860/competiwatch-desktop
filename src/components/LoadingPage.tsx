import React from "react";
import LayoutChildrenContainer from "./LayoutChildrenContainer";
import Blankslate from "./Blankslate";

const LoadingPage = () => (
  <LayoutChildrenContainer>
    <Blankslate>
      <h1>
        <span className="ion ion-md-refresh mr-3 ion-spin" />
        Loading...
      </h1>
    </Blankslate>
  </LayoutChildrenContainer>
);

export default LoadingPage;
