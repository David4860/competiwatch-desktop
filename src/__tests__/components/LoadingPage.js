import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import LoadingPage from "../../components/LoadingPage";

it("matches snapshot", () => {
  const tree = renderer.create(<LoadingPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
