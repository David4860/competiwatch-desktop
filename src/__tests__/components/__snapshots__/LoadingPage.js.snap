// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="Box-sc-1b6inku-0 sc-AxjAm bAeCJN"
>
  <div
    className="sc-AxirZ eLNlIy"
  >
    <h1>
      <span
        className="ion ion-md-refresh mr-3 ion-spin"
      />
      Loading...
    </h1>
  </div>
</div>
`;
